src/screens/notifications/notifications.tsx

```diff
@@ -1,29 +1,27 @@
+import { FlashList, ListRenderItem } from '@shopify/flash-list';
 import React, { useEffect } from 'react';
-import { FlatList, ListRenderItem } from 'react-native';
 import { useDispatch } from 'react-redux';
 
-import { DataPlaceholder } from '../../components/data-placeholder/data-placeholder';
-import { useScreenContainerStyles } from '../../components/screen-container/screen-container.styles';
-import { NotificationInterface } from '../../interfaces/notification.interface';
-import { ScreensEnum } from '../../navigator/enums/screens.enum';
-import { viewAllNotificationsAction } from '../../store/notifications/notifications-actions';
-import { useNotificationsSelector } from '../../store/notifications/notifications-selectors';
-import { usePageAnalytic } from '../../utils/analytics/use-analytics.hook';
+import { DataPlaceholder } from 'src/components/data-placeholder/data-placeholder';
+import { NotificationInterface } from 'src/interfaces/notification.interface';
+import { ScreensEnum } from 'src/navigator/enums/screens.enum';
+import { viewAllNotificationsAction } from 'src/store/notifications/notifications-actions';
+import { useNotificationsSelector } from 'src/store/notifications/notifications-selectors';
+import { usePageAnalytic } from 'src/utils/analytics/use-analytics.hook';
+
 import { NotificationPreviewItem } from './notification-preview-item/notification-preview-item';
 import { NotificationsStyles } from './notifications.styles';
 
 const VIEW_ALL_NOTIFICATIONS_TIMEOUT = 5 * 1000;
+const AVERAGE_NOTIFICATION_ITEM_HEIGHT = 127;
```

Comment:
Did you mean `Math.floor(formatSize(127))`?

Reply:
look at the 1 comment thread